src/assets/small-icons/index.js

```diff
@@ -97,6 +98,7 @@ export { default as starBorderIcon } from './ic_star-border.svg';
 export { default as starFilledIcon } from './ic_star-filled.svg';
 export { default as starHalfFilledIcon } from './ic_star-half-filled.svg';
 export { default as tasksIcon } from './ic_tasks.svg';
+export { default as thumbUpIcon } from './ic_thumb-up.svg';
```

Comment:
This one has also been added but not listed in the PR description. I don't know where it is used but can definitely be replaced afterwards.

Reply:
I updated the description.